assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/useActions.ts

```diff
@@ -18,7 +19,8 @@ const useActions = ({ ticketId }) => {
 
 	const { id, cacheId } = ticket;
 
-	const { createEntity, deleteEntity } = useTicketMutator(id);
```

Comment:
when I added `deleteEntity` here was thinking if that was a good approach, since `id` would have been provided later and passing id to `useTicketMutator` doesn't had a lot of sense